test/include-test.bats

```diff
@@ -12,3 +12,7 @@
   [[ -n $(type -t __bp_install) ]]
 }
 
+@test "bp should stop installation if HISTTIMEFORMAT is readonly" {
+  readonly HISTTIMEFORMAT
+  ! source "${BATS_TEST_DIRNAME}/../bash-preexec.sh"
```

Comment:
Could you add assert the logging is working as well? e.g.
```
  [[ "$output" == "bash-preexec requires write access to HISTTIMEFORMAT" ]]
```

Reply:
Done.